benchmarking/smartseq2/combine_QCs/combine_QC_quant_metrics.R

```diff
@@ -0,0 +1,76 @@
+# R script to do metrics test 
```

Comment:
spell check this.

Reply:
Additionally, can we decide on a style guide and use it to format the R code? I particularly like Google's guides: 

https://google.github.io/styleguide/Rguide.xml

Generally, because I am less familiar with R, there are a few things that would really help me to review: 

1. More comments: it would be nice to get full docstrings (https://google.github.io/styleguide/Rguide.xml#functiondocumentation) for functions including comments on inputs. 
2. Please use keyword arguments whenever possible, as it helps make magic numbers like `axis` parameters in apply more interpretable. 
3. In cases where there are magic numbers (like you are selecting columns `c(1,2)` it would be nice to know what those columns are
4. When using data frames, referencing the columns by name would be very helpful. 
5. When creating variable names, take advantage of them by using full variable names to provide as much information as possible. 

I really want to review the science of this code but I feel it requires too much expert knowledge about the data that I don't have. As with the other PR, I'll be very happy to review this again if you can help me understand by working through my above suggestions. 

If it would help, I'd be happy to walk through the code with you and point out where I'm having trouble.